src/com/google/enterprise/adaptor/filenet/DocumentTraverser.java

```diff
@@ -86,25 +87,58 @@
       Logger.getLogger(DocumentTraverser.class.getName());
 
   private final ConfigOptions options;
-  private final int maxRecords;
+  private Checkpoint incrementalCheckpoint;
 
   public DocumentTraverser(ConfigOptions options) {
     this.options = options;
-    // Leave room for the continuation URLs.
-    this.maxRecords = options.getMaxFeedUrls() - 2;
+    Date yesterday = new Date(new Date().getTime() - TimeUnit.DAYS.toMillis(1));
+    this.incrementalCheckpoint =
+        new Checkpoint("incremental", yesterday , Id.ZERO_ID);
   }
 
   @Override
   public void getDocIds(Checkpoint checkpoint, DocIdPusher pusher)
       throws IOException, InterruptedException {
+    ArrayList<DocIdPusher.Record> records = new ArrayList<>();
+    // - 2 to leave room for the continuation URLs.
+    Checkpoint newCheckpoint =
+        getDocIds(checkpoint, false, records, options.getMaxFeedUrls() - 2);
+    // TODO(jlacey): if (records.size() == maxRecords)
```

Comment:
Why leave this as a TODO when you're using this condition for getModifiedDocIds? As a nit, maxRecords doesn't exist here anymore. Could declare a local variable, I guess, especially if we implement this TODO and use it twice.